index.html

```diff
@@ -11,7 +11,23 @@
 
 </head>
 
+<header>
+    <img class="lambda_logo" src="img/lambda-black.png" alt="lambda-black">
+    <nav>
+        <a href="index.html">Home</a>
```

Comment:
It's good to see that you properly linked to both the about.html and index.html page. Many people overlook this and as a result users can get lost while navigating their page